shell/platform/android/io/flutter/Build.java

```diff
@@ -8,6 +8,7 @@
 public class Build {
   /** For use in place of the Android Build.VERSION_CODES class. */
   public static class API_LEVELS {
+    public static final int FLUTTER_MIN = 21;
```

Comment:
We should add some documentation for why this value exists and add pointers to it from the other places we change the minimum value. Also I think we should make this value visible for testing.  I think in production/shipped code we want to set an actual bound and if minimum value gets higher than a split review that code and remove one branch. But for tests this lets us say "this method should support all api versions that flutter supports". 

https://github.com/flutter/flutter/blob/b5749a7d65be14d655a3de4fe9d82cc758c3b85e/docs/platforms/android/New-Android-version.md?plain=1 is a good starting point for finding where we might need to link to and from this file. 

// The minimum android api that flutter supports. Intended for use in tests that don't actually care about the api level but want to set a minSdk bound that does not need to regularly be updated when the minimum version changes.

Reply:
Doc is located in the flutter/flutter repo. Will make a different PR to update the docs.